src/styles/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__lable {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main_color);
+  padding-top: 40px;
+}
+
+.card__code,
+.review__text,
+.card__price-lable {
+  font-weight: 400;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary_color);
+  padding-top: 4px;
+}
+
+.review__stars {
```

Comment:
in case you DO need `review` BEM-block, make sure to put its styles into another dedicated file

Reply:
Thank you. followed your comments and made up a new solution